inspire/base/templates/accounts/settings/helpers.html

```diff
@@ -0,0 +1,58 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+ {%- extends "accounts/settings/helpers_base.html" -%}
+
+{%- macro panel_start(title, icon="", btn_text='', btn='', btn_icon='', btn_href='', btn_class='btn-default', btn_name='', id="", with_body=True) %}
+{%- block panel_start scoped %}
+<div class="panel-group panel-bot-margin"{% if id %} id="{{id}}"{% endif %}>
+    <div class="panel panel-primary">
+        <div class="panel-heading">
+            <strong>{%if icon %}<i class="{{icon}}"></i>{% endif %} {{title}}</strong>
+            {%- if btn and (btn_href or btn_text) -%}
+            <div class="pull-right">
+            {%- if btn_text %}
+                <small class="text-muted small-style">(updated {{last_sync}})</small>
```

Comment:
A class name ``name-style`` it's not the best of namings. Can you have a look at https://css-tricks.com/semantic-class-names/ for reference of good css class naming?